monitoring_service.rst

```diff
@@ -4,36 +4,24 @@ Monitoring Service
-*   Then, whenever a channel is closed by calling ``closeChannel`` and if the client did not react within a specified
-    timeout, the MS will call ``updateNonClosingBalanceProof`` with the submitted MR on behalf of its client.
-*   Rewards are paid in RDN, there is no free tier.
-*   In the current stage, this is a simple design which is expected to help to reach the Ithaca milestone earlier
-*   In the long-term a PISA approach seems more economically viable and user friendly, but this design requires
-    additional features and can be developed independently after Ithaca.
-
-Usual Scenario
-==============
+Overview
+========
+
+Monitoring services watch open payment channels when the user is not online. In
+case one channel partner closes a channel while the counterparty is offline (or
+doesn’t react for 80% of the settlement timeout after close has been called),
+the monitoring service sends the latest balance proof to the channel smart
+contract and thus ensures correct settlement of the channel.
+
+To do this a Monitoring Service (MS) listens to :ref:`Monitor Requests <Monitor
+Request>` in a public Matrix room. An MR is accompanied by a reward for acting
+on it. Based on this reward, the MS can decide to monitor a channel and store
+the corresponding MRs.
 
-The Raiden node that belongs to Alice is going offline and Alice wants to be protected against having her channels
-closed by Bob with an incorrect balance proof.
-
-1)  Whenever Alice wants to get her channel monitored, Alice publishes her `Monitor Request`_ - including a blinded balance
-    proofs - into a public chat room
-2)  Updates can be sent by Alice whenever the state or offered reward changes
-3)  MSs can pick up these messages and then listen to ``ChannelClosed`` events regarding this particular channel
-4)  If so, whenever the channel is closed and before the settlement period has ended, the MS will call the
-    ``updateNonClosingBalanceProof`` function with the provided information, after a period of x blocks where the client
-    is expected to react. This way the settleChannel function that calculates the token distribution can only be
-    executed submitting the corresponding balance values of the hashed balances provided before
-5)  Either Alice or Bob (or anyone else) can now anytime call ``settleChannel`` and initiate the token distribution
-6)  After a successful submission of the hashed balance by the MS calling ``updateNonClosingBalanceProof`` (might be a
-    defeat of an attack), the monitoring service gets an on-chain payment from a smart contract where Alice deposited
-    some tokens beforehand
-7)  This will result in a correct distribution of tokens, in accordance with the off-chain transfers
+Whenever a channel is closed by calling ``closeChannel`` and if the client did
+not react itself, the MS will call ``updateNonClosingBalanceProof`` with the
+submitted MR on behalf of its client. For that action then MS can then claim the
```

Comment:
I see one too many `then`.  Maybe drop the first one?

Reply:
Thanks. This should be an `the`.